src/shared-components/field/style.ts

```diff
@@ -17,7 +20,7 @@ export const FieldContainer = styled.div`
   margin-bottom: 22px;
 `;
 
-export const Label = styled.label`
+export const Label = styled.label<{ disabled?: boolean }>`
```

Comment:
There might've been a typing error before, but since we're relying on default props in a function component now this prop is always being passed to Label

Reply:
fixed, yeah this is another one of the things I want to review and standarize after all is converted